src/components/App/index.test.tsx

```diff
@@ -12,5 +12,5 @@ it('renders without crashing', () => {
   const root = shallow(<App />, { context: { store } }).shallow();
 
   expect(root).toHaveClassName(styles.container);
-  expect(root.find(styles.header)).toHaveLength(1);
+  expect(root.find(`.${styles.header}`)).toHaveLength(1);
```

Comment:
Because the test was not accurate to begin with - it was a false positive. `styles.header` returns the string `header`, which yields `root.find(header)`, which worked before because the element in question was a `header` element. I believe what we wanted to yield is `root.find('.header')`, meaning "find the element with the *class* called `header`". That's what the new code is doing. This came to light because I am using a `Row` element now rather than a `header` element.

Reply:
huh? Was it failing?

With CSS modules, we should keep what it was before because `styles.header` is likely not `.header` but rather `.App-header-xy123`.

Reply:
👍

Reply:
Oh, nice. I was confused by this before.